letter-shop.md

```diff
@@ -0,0 +1,255 @@
+# Letter Shop Tutorial
+
+## What you need before you start:
+
+* a laptop with an internet connection and NodeJS installed
+* basic knowledge of the command line terminal
+* basic knowledge of JavaScript, including Promises and Buffers
+* (optional) the concept of one-way hash functions
+
+## What you'll learn:
+
+* the concept Hash Time Lock Agreements (HTLAs)
+* the Ledger Plugin Interface (LPI)
+* how to build a paid web app using Interledger!
+
+## Step 1: Creating the Letter Shop
+
+Save the following JavaScript as `shop.js`:
```

Comment:
It should start with the instructions for cloning and `npm install`ing the repo instead

Reply:
OK. An advantage of that would be that we can then discuss shorter snippets of the code and not have to paste entire files - especially where shop.js and shop2.js and shop3.js from the next tutorial have a lot in common.